server/db/seeds/11-gardenShop.js

```diff
@@ -0,0 +1,12 @@
+exports.seed = async function (knex) {
+  // Deletes ALL existing entries
+  await knex('garden_shop').del()
+  await knex('garden_shop').insert([
+    {
+      id: 1,
+      gardens_id: 3,
+      shop_products_id: 2,
+      stock: 15,
+    },
```

Comment:
Would it make sense to include a couple more entries to prove that your db join works as intended?  eg:

{
      id: 2,
      gardens_id: 3,
      shop_products_id: 1,
      stock: 10,
},
{
      id: 3,
      gardens_id: 2,
      shop_products_id: 1,
      stock: 22,
}